src/test/unittest/granularity.py

```diff
@@ -192,13 +192,16 @@ class _Granularity(Enum):
 ANY = _Granularity.ANY
 
 
-def require_granularity(granularity, **kwargs):
-    if not isinstance(granularity, _Granularity):
-        raise ValueError('selected granularity {} is invalid'
-                         .format(granularity))
+def require_granularity(*granularity, **kwargs):
```

Comment:
All cases with required 1 granularity stopped working (silent skip).
Cases with 2 granularities do not work also, but it is possible that I can't use it :)